src/components/Activity/ActivitiesInstance/ActivitiesInstance.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import '../../../css/conversationsActivitiesInstanceShared.css';
+import ConversationsMessageAvatar from '../../Conversations/ConversationsMessageAvatar/ConversationsMessageAvatar.jsx';
+import ActivitiesInstanceText from '../ActivitiesInstanceText/ActivitiesInstanceText.jsx';
+import ActivitiesInstanceButtons from '../ActivitiesInstanceButtons/ActivitiesInstanceButtons.jsx';
+
+export default class extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    formatTime(start_time, end_time) {
+        return start_time.substring(0, 5) + '-' + end_time.substring(0, 5);
+    }
+
+    formatDate(start_date, end_date, frequency) {
+        const WEEKDAYS = ['man', 'tirs', 'ons', 'tor', 'fre'];
+        let workdays = '';
+        let formattedDate = '';
+        if (start_date === end_date) {
+            const date = new Date(start_date);
+            const month =
```

Comment:
Det finnes nok andre best practices for å ta ut måneder, men her med denne måten kunne du kanskje bare laget en metode for gjenbruken da det blir repetert ganske mange terniarys her. Bare for leselighets skyld. Ikke nødvendig nå, men fint å tenke på.

Reply:
Ingen annelse hva som skjer her tbh

Reply:
Har ikke du skrevet det da?

Reply:
Mesteparten er fra junior

Reply:
aiaiai, muy caramba

Reply:
Hahaha :stuck_out_tongue: Hvis dere har spørsmål angående det junior har laget så har de sagt at det bare er å spørre

Reply:
Haha, skal spørre Casper senere :)